docs/api/api_blueprint.apib

```diff
@@ -21379,35 +21379,41 @@ Create a new feedback with event_id.
         {
             "data": {
                 "relationships": {
-                    "event": {
+                    "session": {
                         "links": {
-                            "self": "/v1/feedbacks/1/relationships/event",
-                            "related": "/v1/feedbacks/1/event"
+                            "self": "/v1/feedbacks/2/relationships/session",
+                            "related": "/v1/feedbacks/2/event"
                         }
                     },
                     "user": {
                         "links": {
-                            "self": "/v1/feedbacks/1/relationships/user",
-                            "related": "/v1/feedbacks/1/user"
+                            "self": "/v1/feedbacks/2/relationships/user",
+                            "related": "/v1/feedbacks/2/user"
+                        }
+                    },
+                    "event": {
+                        "links": {
+                            "self": "/v1/feedbacks/2/relationships/event",
+                            "related": "/v1/feedbacks/2/event"
                         }
                     }
                 },
                 "attributes": {
-                    "rating": "4",
+                    "rating": 4.0,
                     "deleted-at": null,
-                    "comment": "Awesome event"
+                    "comment": "Awesome session."
                 },
                 "type": "feedback",
-                "id": 1,
+                "id": 2,
```

Comment:
Since there is no other feedback in system before POST request, this should be 1?